Examples/NewModuleTemplate/plasterManifest.xml

```diff
@@ -53,6 +53,10 @@
         </parameter>
     </parameters>
     <content>
+        <message>
+Scaffold a PowerShell Module with the files required to run Pester tests, build with PSake and publish to the PSGallery.
+
```

Comment:
So the whitespace is not being removed from the beginnings of the "Here's" lines?  Or is the XML parser taking care of that?  Either way, sounds good to me!